app/views/shared/_micropost_form.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= form_for(@micropost) do |f| %>
+  <%= render 'shared/error_messages', object: f.object %>
+  <div class="field">
+    <%= f.text_area :content, placeholder: t("micropost.placeholder") %>
+  </div>
+  <%= f.submit t("micropost.post"), class: "btn btn-primary" %>
+  <span class="image">
+    <%= f.file_field :image, accept: Settings.files.valid_input %>
+  </span>
+<% end %>
+
+<script type="text/javascript">
+  $("#micropost_image").bind("change", function() {
+    var size_in_megabytes = this.files[0].size/1024/1024;
+    if (size_in_megabytes > 5) {
+      alert( <%= t("micropost.error.alert")%> );
+    }
+  });
+</script>
```

Comment:
Do not write js in html file, you should write to another js file and required to application.js